src/main/java/org/openlmis/stockmanagement/controller/StockCardController.java

```diff
@@ -38,11 +51,35 @@
 @Controller
 @Api(value = "Stock Cards", description = "Track the stock cards (stock on hand) at various facilities.")
 @RequestMapping(value = "/api/v2/")
+@NoArgsConstructor
 public class StockCardController extends BaseController
 {
+    private static Logger logger = Logger.getLogger(StockCardController.class);
+
+    @Autowired
+    private FacilityRepository facilityRepository;
+
+    @Autowired
+    private ProductService productService;
```

Comment:
What is the reasoning for using the product service, but using the facility and stock adjustment reason repos? When should we include services and when should we include repos?